lib/src/options.dart

```diff
@@ -157,7 +157,7 @@ class VideoPublishOptions {
   final List<VideoParameters> screenShareSimulcastLayers;
 
   const VideoPublishOptions({
-    this.videoCodec = 'H264',
+    this.videoCodec = 'vp8',
```

Comment:
does this change need to be communicated to users in a new release?

Reply:
good call, we should include this in the release notes. My guess is Duan is changing this for compatibility reasons?

Reply:
Yes, this modification should have nothing to do with this PR, I forgot to revert it after testing.   :)